src/main/java/frc/robot/Constants.java

```diff
@@ -18,14 +20,23 @@ public static class MotorIDConstants {
     public static final int kElevatorMotor1 = 1;
     public static final int kElevatorMotor2 = 2;
     public static final int kIntakeMotor = 9;
+    public static final int kPivotMotor = 0; // FIXME: Change to correct ID
```

Comment:
This one is a CTRE motor, so it will be on the CANivore Bus (mispelled as CANavor below). Probably we should make separate constants classes for each